js/src/setup-mc/setup-stepper/setup-paid-ads/setup-paid-ads.js

```diff
@@ -19,19 +22,71 @@ import AppButton from '.~/components/app-button';
 import ProductFeedStatusSection from './product-feed-status-section';
 import PaidAdsFeaturesSection from './paid-ads-features-section';
 import GoogleAdsAccountSection from './google-ads-account-section';
+import AudienceSection from '.~/components/paid-ads/audience-section';
+import BudgetSection from '.~/components/paid-ads/budget-section';
+import validateForm from '.~/utils/paid-ads/validateForm';
 import { getProductFeedUrl } from '.~/utils/urls';
 import { GUIDE_NAMES } from '.~/constants';
 import { API_NAMESPACE } from '.~/data/constants';
 
-function PaidAdsSectionsGroup() {
-	// TODO: Add audience and budget sections.
-	return <GoogleAdsAccountSection />;
+function PaidAdsSectionsGroup( { onCampaignChange } ) {
+	const { googleAdsAccount } = useGoogleAdsAccount();
+	const { data: targetAudience } = useTargetAudienceFinalCountryCodes();
+
+	if ( ! targetAudience ) {
+		return <GoogleAdsAccountSection />;
+	}
+
+	const handleChange = ( _, values, isValid ) => {
+		onCampaignChange( { ...values, isValid } );
+	};
+
+	return (
+		<Form
+			initialValues={ {
+				amount: 0,
+				countryCodes: targetAudience,
+			} }
+			onChange={ handleChange }
+			validate={ validateForm }
+		>
+			{ ( formProps ) => {
+				const { countryCodes } = formProps.values;
+				const disabledAudience =
+					googleAdsAccount?.status !== 'connected';
```

Comment:
💅  This status could be also defined as Constant I guess

Reply:
Updated in https://github.com/woocommerce/google-listings-and-ads/pull/1655/commits/4d7da2b6bd972f1989b93c4cb23d6ec0d4fff171.